app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -3358,7 +3361,7 @@ private void onTextTracksChanged() {
         final List<String> availableLanguages = new ArrayList<>(textTracks.length);
         for (int i = 0; i < textTracks.length; i++) {
             final TrackGroup textTrack = textTracks.get(i);
-            if (textTrack.length > 0 && textTrack.getFormat(0) != null) {
```

Comment:
what was the warning here? can never be null?

Reply:
Yes, probably a `@NonNull` was added by exo player